src/main/java/uk/gov/hmcts/reform/sscs/service/VenueService.java

```diff
@@ -24,13 +24,17 @@ public String getEpimsIdForVenue(String processingVenue) {
         String venueId = String.valueOf(airLookupService.getLookupVenueIdByAirVenueName()
             .get(processingVenue));
 
+        return getEpimsIdForVenueId(venueId);
+    }
+
+    public String getEpimsIdForVenueId(String venueId) {
         VenueDetails venueDetails = venueDataLoader.getVenueDetailsMap()
             .get(String.valueOf(venueId));
 
         return Optional.ofNullable(venueDetails)
             .map(VenueDetails::getEpimsId)
             .orElseThrow(() -> {
-                throw new IllegalStateException("Unable to find epims ID for processing venue: " + processingVenue);
+                throw new IllegalStateException("Unable to find epims ID for venue ID: " + venueId);
```

Comment:
These changes have been reverted in the base branch 11092. This was done to keep features separate, as these belong to @nickhill111 ticket 10293.